elisp/gilded-rose.el

```diff
@@ -0,0 +1,37 @@
+(defun make-item (name sell-in quality)
+  "Create an item with NAME, SELL-IN, and QUALITY."
+  (list :name name :sell-in sell-in :quality quality))
+
+(defun update-quality (items)
+  (dolist (item items)
+    (if (and (not (string= (plist-get (car item) :name) "Aged Brie"))
```

Comment:
I do not now elisp. Is (car item) necessary when you already do (dolist (item items)) so item is an item with we access with plist-get, right?

Reply:
That's a good point. I thought `car` was needed to get the name as the first element of an `item`. I'll remove it and run my tests against the code

Reply:
@codecop you were right. The `car` is not needed